protocol/messenger_test.go

```diff
@@ -2484,6 +2484,22 @@ func (t *testTimeSource) GetCurrentTime() uint64 {
 	return uint64(time.Now().Unix())
 }
 
+func (s *MessengerSuite) TestHandleStandaloneChatIdentity() {
+	chat := CreatePublicChat("test-public-chat", s.m.transport)
+	err := s.m.SaveChat(chat)
+	s.NoError(err)
+
+	err = s.m.handleStandaloneChatIdentity(chat)
+	s.NoError(err)
+
+	chat = CreateOneToOneChat("test-private-chat", &s.privateKey.PublicKey, s.m.transport)
+	err = s.m.SaveChat(chat)
+	s.NoError(err)
+
+	err = s.m.handleStandaloneChatIdentity(chat)
+	s.NoError(err)
```

Comment:
Here we could add an assert to check that the large image is there

Reply:
> Here we could add an assert to check that the large image is there

That's right, working on a proper test here.